pkg/networkserver/networkserver_flow_test.go

```diff
@@ -413,6 +414,7 @@ func handleClassAOTAAEU868FlowTest1_0_2(ctx context.Context, conn *grpc.ClientCo
 					Key: &appSKey,
 				},
 				SessionKeyID: []byte("session-key-id"),
+				ReceivedAt:   asUp.GetJoinAccept().GetReceivedAt(),
```

Comment:
@rvolosatovs do you see a way of achieving this without simulating the wall clock ? (we could do that in this test as well, if you consider it wouldn't be too artificial).

Reply:
This looks good to me - I don't we should mock time in flow tests.